modules/fb_instant_articles_display/src/InstantArticle.php

```diff
@@ -69,19 +68,20 @@ class DrupalInstantArticleDisplay {
    * @param $instantArticle
    */
   private function __construct($node, $layoutSettings, $instantArticle) {
+    parent::__construct();
```

Comment:
@m4olivei  Actually it doesn't now… with my changes `return new InstantArticle($node, $layoutSettings, $instantArticle)` instantiates a version of itself now in `create()`. How about we set the vars right within `create()` and then just remove `__construct()`?